document_merge_service/api/views.py

```diff
@@ -48,17 +51,36 @@ def merge(self, request, pk=None):
         convert = serializer.data.get("convert")
 
         if convert:
-            url = f"{settings.UNOCONV_URL}/unoconv/{convert}"
-            requests_response = requests.post(url, files={"file": response.content})
-            fmt = settings.UNOCONV_FORMATS[convert]
-            extension = fmt["extension"]
-            content_type = fmt["mime"]
-
-            response = HttpResponse(
-                content=requests_response.content,
-                status=requests_response.status_code,
-                content_type=content_type,
-            )
+            if settings.UNOCONV_LOCAL:
+                filepath = Path(f"/tmp/{uuid4()}.docx")
```

Comment:
What's the reason for not using `tempfile`? for example `tempfile.NamedTemporaryFile(suffix='.docx')`

Reply:
The best reason there is: I didn't know about it :)

Thanks for letting me know!